docs/manufacturer/connector-standard.mdx

```diff
@@ -43,6 +43,13 @@ An additional 2-pin connector for power (ext. power) can be used for high-powere
 
 We recommend using twisted wires to eliminate any confusion about the mirroring of the connector and to ensure that the same wiring order is used on both sides of the connection.
 
+V+ connection from the ESC to FC will typically carry VBAT voltage direct from the battery connection. The ESC V+ connection is an Input Voltage to the FC whilst RX, GPS and other connectors' 
+V+ pads carry output from the FC's onboard voltage regulators. 
+
+In some cases VTX or camera connectors may offer VBAT voltage directly but due to voltage fluctuations induced by the motors the use of VBAT direct to VTXs or cameras is discouraged. 
+To minimise the risk to sensitive VTX hardware it is advisable to provide an additional high voltage regulator for such components. 
```

Comment:
```suggestion
To minimize the risk to sensitive VTX hardware it is advisable to provide an additional high voltage regulator for such components. 
```
Looked it up:
![image](https://github.com/betaflight/betaflight.com/assets/8344830/f1dd1893-f1ce-44e1-b4a8-3e9319de2493)

Reply:
Yes, I'm a Brit. Do we have a particular standard for this? I don't think any meaning is lost.

Reply:
Counting 23 x `minimise` and 13 x `minimize` in docs folder :)

Reply:
Ok. Leaving it as "ise" which is also the Australian spelling

Reply:
Yea we speak kings English here not American